File: tests/go/cases/case_144_indirect_container_write_05.go
package escape_tests

var retainedCase144 = []map[string]string{}

func Case144IndirectContainerWrite05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "indirect_container_write_05",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "indirect_container_write_05:" + raw,
	}
	envelope := map[string]map[string]string{"wrapped": payload}
	// ESCAPE: indirection writes payload into retained sink.
	retainedCase144 = append(retainedCase144, envelope["wrapped"])
	return "ok"
}

File: tests/nodejs/cases/case_144_indirect_container_write_05.js
/**
 * Task case 144: indirect_container_write_05 false-positive/false-negative stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case144IndirectContainerWrite05(input) {
  const taskName = 'indirect_container_write_05';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'stress',
    stage: 'evaluation',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const envelope = { wrapped: payload };
  // ESCAPE: payload stored via indirection into retained sink.
  retainedAudit.push(envelope.wrapped);
  return 'ok';
}

module.exports = {
  case144IndirectContainerWrite05,
};
